src/main/java/com/excella/foodinc/models/Food.java

```diff
@@ -0,0 +1,60 @@
+package com.excella.foodinc.models;
+
+public class Food {
+
+    private String name;
+    private int calories;
+    private int fat;
+    private int protein;
+    private int carbs;
+
+    public Food() {
+
+    }
+    public Food(String name, int calories, int fat, int protein, int carbs) {
+        this.name = name;
+        this.calories = calories;
+        this.fat = fat;
+        this.protein = protein;
+        this.carbs = carbs;
+    }
+    public void setName(String name) {

```

Comment:
You shouldn't even need `set` methods if you just create the foods with the constructor.